### ISSUE TITLE
Meteor dynamic import produces parsing error

### ISSUE THREAD

**Issue by matteodem** [eslint/eslint#8789]
**Tell us about your environment**

* **ESLint Version:** 4.0.0
* **Node Version:** v6.10.3
* **npm Version:** 3.10.10

**What parser (default, Babel-ESLint, etc.) are you using?** eslint-config-standard

**Please show your full configuration:**

```js
module.exports = {
    "extends": "standard",
    "plugins": [
        "standard",
        "promise"
    ],
    "rules": {
        "comma-dangle": "off",
        "spaced-comment": "off"
    }
};
```

**What did you do? Please include the actual source code causing the issue.**

Code like following create a parsing error

```js
import { FlowRouter } from 'meteor/ostrio:flow-router-extra'

FlowRouter.route('/', {
  title: 'Boilerplate Home',
  action() {
    this.render('mainLayout', 'Home')
  },
  waitOn() {
    return import('../components/Home/HomeComponent.js')
  },
})
```

**What did you expect to happen?**

To run through or create a linting error.

**What actually happened? Please include the actual, raw output from ESLint.**

```bash
yarn lint v0.23.2
$ eslint 'client/*.js' 'server/*.js' 'imports/**/*.js'

/Users/Matteo/Projects/meteor/boilerplate/app/imports/ui/routes/MainRoutes.js
  9:12  error  Parsing error: Unexpected token import

✖ 1 problem (1 error, 0 warnings)

error Command failed with exit code 1.
```

**Comment by not-an-aardvark** [comment]
Hi, thanks for the report.

Dynamic import is a [stage 3 proposal](https://github.com/tc39/proposal-dynamic-import), and it's still an experimental feature. We generally don't support parsing experimental features in ESLint (see our [readme](https://github.com/eslint/eslint/blob/e8f1362ab640c883a5d296e951308fab22073e7f/README.md#what-about-experimental-features)).

You can use [babel-eslint](https://github.com/babel/babel-eslint) as a custom parser enable parsing dynamic import.

**Comment by matteodem** [comment]
Thank's for the help, I'll try that then!
